.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,32 @@
+name: build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install go
+        uses: actions/setup-go@v4
+        with:
+          go-version: ^1.20
+
+      - name: Run Trivy vulnerability scanner in repo mode
```

Comment:
We have been thinking about using Github's built-in security scanning for csi repos instead of trivy. Do you see any advantages for using trivy?

Reply:
Not really, I thought we didn't have any sort of visibility on vulns so I copied https://github.com/kubernetes-csi/external-snapshotter/blob/master/.github/workflows/trivy.yaml to this repo but if we already have something built in then yes I think we can use that instead.

Reply:
I removed the file.